src/platform/site-wide/announcements/selectors.js

```diff
@@ -1,4 +1,14 @@
 import _config from './config';
+import moment from 'moment';
+
+function isExpiredAnnouncement(announcement) {
+  if (!announcement.expiresAt) return true;
+
+  const expirationDate = moment(announcement.expiresAt);
+  const isExpired = moment().isSameOrAfter(expirationDate);
+
+  return !isExpired;
+}
```

Comment:
If there isn't an expiration date, then it goes on indefinitely, so just return true. Otherwise, check the current time against the expiration date. If our current time is after, this announcement is expired.